docs/source/examples/best-practices.rst

```diff
@@ -0,0 +1,112 @@
+Best Practices
+==============
+
+
+Multi-GPUs
+~~~~~~~~~~
+
+When using multiple GPUs, if possible, it's best to co-locate as many GPUs on the same physical node.  This could be a
```

Comment:
Do we need to say _how_ to co-locate GPUs? Or do you just mean "When choosing between two multi-GPU setups, it is best to pick the one where most GPUs are co-located with one-another".